src/detector.js

```diff
@@ -1,10 +1,14 @@
 export default class Detector {
 
   static platform() {
-    if (/iPhone|iPad|iPod/i.test(window.navigator.userAgent)) {
```

Comment:
Let's not chain this here in `platform`. I'd call for another static method, e.g. `device` which is only called if tablet or phone specific meta tag is used.